proteus/TwoPhaseFlow/models/rans2p_p.py

```diff
@@ -50,9 +57,11 @@
 # ************************************ #
 # ********** MODEL INDEXING ********** #
 # ************************************ #
-CLSVOF_model=1
-VF_model=None
-LS_model=None
+ME_model = mparams.rans2p['index']
+assert ME_model != None, 'rans2p model index was not set!'
```

Comment:
Change to `is not None` instead of `!=`. We decided some time ago to use that convention. If I recall correctly.

Reply:
Yes it should be `is not None`